src/components/Header.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import Header from './Header'
+import { MemoryRouter } from 'react-router-dom'
+
+test('renders header with logo', () => {
+  const { getByAltText } = render(
+    <MemoryRouter>
+      <Header />
+    </MemoryRouter>
```

Comment:
Do you need MemoryRouter here? I'm not sure.

Reply:
My logo is linked to the home page within my header. That's why I used it :)

Reply:
Ah, understood! Then you're all good to go... 👍